docs/TUTORIAL.md

```diff
@@ -41,160 +47,307 @@ This tutorial uses the following billable components of Google Cloud:
+1. In Cloud Shell, specify the project that you want to use for this tutorial:
+   ```
+   gcloud config set project PROJECT_ID
+   ```
+   Replace PROJECT_ID with the ID of the project that you selected or created for this tutorial. 
+
+   If a dialog displays, click *Authorize*.
+
+
+2. Specify a default region to use for infrastructure creation:
+   ```
+   gcloud config set compute/region REGION
+   ```
+
+3. Create and activate a Python virtual environment:
+   ```
+   python -m venv reg-rpt-env
+   source reg-rpt-env/bin/activate
+   ```
+
+   You see that your command-line prompt is prefixed with the name of the virtual environment.
+
+
+4. Clone the gitHub repository to your development environment
+   ```
+   git clone "https://github.com/GoogleCloudPlatform/reg-reporting-blueprint"
+   ```
+
+
+5. Install Terraform. To learn how to do this installation, see the [HashiCorp documentation](https://learn.hashicorp.com/tutorials/terraform/install-cli#install-terraform).
+
+
+6. [Verify](https://learn.hashicorp.com/tutorials/terraform/install-cli#verify-the-installation) the installation 
+
+
+7. Install  dbt:
     ```
     sudo pip3 install dbt-bigquery
     ```
 
-    To verify the installation of dbt, execute the following command:
+
+8. Verify the dbt installation:
     ```
     dbt --version
     ```
 
-    You see a printed version of the installation.
+    You see the installation details displayed.
 
-2. Initialize the environment variables: 
-    ```
-    cd pattern-solution
-    source environment-variables.sh
-    ```
 
-3. Run the setup script.
+9. Initialize the environment variables: 
     ```
-    cd common_components
-    . ./setup_script.sh
+    cd reg-reporting-blueprint && source environment-variables.sh
     ```
 
-4. Run terraform to create the required infrastructure
-    ```
-    cd orchestration/infrastructure/
-    terraform init -upgrade
-    terraform plan
-    terraform apply
-    ```
 
-5. In the Google Cloud Console, go to the **Cloud Storage** page and check for a bucket with a name like ${project}-${region}-ingest-bucket to verify that an ingest bucket has been created.
+10. Run the setup script.
+     ```
+     cd common_components && ./setup_script.sh
+     ```
+    
+    This will create a `backend.tf` and `terraform.tfvars` files based on the templates.
+    If you wish to create a composer infrastructure, manually amand the`terraform.tfvars` so that it 
+has `enable_composer=true`  
+
+
+11. Run terraform to create the required infrastructure
+     ```
+     cd orchestration/infrastructure/
+     terraform init -upgrade
+     terraform plan
+     terraform apply
+     ```
+
+    If you enabled Composer at the step before, you will see a URL for the airflow ui. Click on the link to verify 
+    the installation.
+
+
+13. In the Google Cloud Console, go to the **Cloud Storage** page and check for a bucket with a name like 
+${project}-${region}-ingest-bucket to verify that an ingest bucket has been created.
```

Comment:
This conflicts now with Github's new feature on math in markdown:
https://github.blog/2022-05-19-math-support-in-markdown/

It looks like the dollar sign needs to be escaped \$ now.

Reply:
ok